### PR TITLE
Add Scottish colour and scotland service to fonts

### LINKED ISSUE #2121: Add Scotland service to fonts and their colour
**Is your feature request related to a problem? Please describe.**
The /scotland service will require fonts and a new colour.

**Describe the solution you'd like**
Add the 'scotland' service to font-families.  Add 'dark saltire' to the colour palette.

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by jimjohnsonrollings**
Resolves #2121

**Overall change:**
Adds the 'scotland' service to font-families, and the dark saltire colour.

**Code changes:**

- Add the 'scotland' service to font-families
- Add the 'dark saltire' colour
- Update colour unit test

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by sareh** [comment]
Looks great so far. You'll want to update the package version in the [`packages/utilities/psammead-styles/package.json`](https://github.com/bbc/psammead/tree/latest/packages/utilities/psammead-styles/package.json) and [`package-lock.json`](https://github.com/bbc/psammead/tree/latest/packages/utilities/psammead-styles/package-lock.json) and add a new row at the top of the [`CHANGELOG.md`](https://github.com/bbc/psammead/tree/latest/packages/utilities/psammead-styles/CHANGELOG.md) file. It will be a minor increase since you're adding new exports, but the package as a whole has backward compatibility. (Right now, version 2.3.0)

**Review by sareh — APPROVED** [review]
👍 Looks great.  

The reason Code Climate is erroring, even though you have added a test for the colour being added, is that none of the unit tests for `colours.js` are currently sufficient. I have opened an issue here to fix the `colours` unit tests: https://github.com/bbc/psammead/issues/2140 . I think that this PR should not need to wait until that is done to be merged in.

**Review by jroebu14 — APPROVED** [review]

**Comment by staylos92** [comment]
Happy for this to be merged 👍

**Pull request merged**
